pkg/providers/gcp/provider_redis.go

```diff
@@ -23,7 +24,10 @@ import (
 
 const (
 	redisInstanceNameFormat = "projects/%s/locations/%s/instances/%s"
+	redisMemorySizeGB       = 16
```

Comment:
After some discussion this morning we've decided to drop this to 1 GiB with no read replicas for the time being. If this needs to be increased it will be discovered during performance testing.